pages/dashboard.vue

```diff
@@ -40,7 +40,15 @@
   </div>
 </template>
 <script>
-export default {}
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+
+export default {
+  asyncData(context) {
+    if (!sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)) {
+      context.redirect('/signin')
```

Comment:
Did you try adding this redirect in the router similar to PPR? That way it'll do it for every page in the app. Might not work in nuxt, but worth a try I think!

Reply:
Not all of the site is protected right now. I think only the dashboard is. And Nuxt handles routes differently (automatically generated). It's possible to extend or add dynamic routes, I'm still looking at this and need to test with the generated site.